### ISSUE TITLE
composer install error, old package names used in ext_emconf.php

### ISSUE THREAD

**Issue by medarob** [carsten-walther/image_copyright#7]
Composer package was found but there are errors because of old package names in `ext_emconf.php`

```
In PackageManager.php line 1110:
                                                                                                                           
  [UnexpectedValueException] The package "image_copyright" depends on "typo3/backend" which is not present in the system.  
```
                                                                                                                        
```
    'constraints' => [
        'depends' => [
            'typo3' => '10.4.99-12.4.99',
            'typo3/core' => '10.4.99-12.4.99',
            'typo3/extbase' => '10.4.99-12.4.99',
            'typo3/fluid' => '10.4.99-12.4.99',
            'typo3/frontend' => '10.4.99-12.4.99',
            'typo3/backend' => '10.4.99-12.4.99',
        ]
    ],
```
it should probably be renamed to this:

```
            'typo3/cms' => '10.4.99-12.4.99',
            'typo3/cms-core' => '10.4.99-12.4.99',
            'typo3/cms-extbase' => '10.4.99-12.4.99',
            'typo3/cms-fluid' => '10.4.99-12.4.99',
            'typo3/cms-frontend' => '10.4.99-12.4.99',
            'typo3/cms-backend' => '10.4.99-12.4.99',
```

**Comment by medarob** [comment]
`'typo3/cms' => '10.4.99-12.4.99',` can be removed too I think.

**Comment by carsten-walther** [comment]
fixed in version 12.4.3
